x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/RealmSettings.java

```diff
@@ -50,6 +51,8 @@
     static final Setting<String> TYPE_SETTING = Setting.simpleString("type", Setting.Property.NodeScope);
     static final Setting<Boolean> ENABLED_SETTING = Setting.boolSetting("enabled", true, Setting.Property.NodeScope);
     static final Setting<Integer> ORDER_SETTING = Setting.intSetting("order", Integer.MAX_VALUE, Setting.Property.NodeScope);
+    static final Setting<List<String>> LOOKUP_REALMS_SETTING =
+            Setting.listSetting("lookup_realms", Collections.emptyList(), Function.identity(), Setting.Property.NodeScope);
```

Comment:
I think we really need to consider the name of this setting.
`lookup_realm` makes sense to us from an implementation point of view, but it's probably not the right name for a setting that is used by ES admins.